internal/packages/packages.go

```diff
@@ -19,6 +19,9 @@ const (
 	// PackageManifestFile is the name of the package's main manifest file.
 	PackageManifestFile = "manifest.yml"
 
+	// PackageChangelogFile is the name of the package's changelog file.
+	PackageChangelogFile = "changelog.yml"
```

Comment:
Could you please put the changelog logic in a separate package, e.g. `internal/packages/changelog`?